<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Materias Primas</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{background-color: rgb(6, 191, 249);}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="inputMethodHints">
      <set>Qt::InputMethodHint::ImhPreferNumbers</set>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_detalle_mp">
      <attribute name="title">
       <string>Materias primas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3">
         <item>
          <widget class="QLineEdit" name="le_buscar_mp">
           <property name="placeholderText">
            <string>Buscar por nombre...</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="tv_mat_primas">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btn_ver_detalle_mp">
           <property name="text">
            <string>Ver</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cerrar_listado_mp">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(22, 183, 40);</string>
           </property>
           <property name="text">
            <string>Salir</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_nuevo_mp">
      <attribute name="title">
       <string>Nueva</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_nombre_nueva">
               <property name="inputMethodHints">
                <set>Qt::InputMethodHint::ImhNone</set>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="clearButtonEnabled">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="lbl_proveedor_nueva">
               <property name="text">
                <string>Proveedor</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_nombre_nueva">
               <property name="text">
                <string>Nombre</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="cb_proveedor_nueva">
               <property name="inputMethodHints">
                <set>Qt::InputMethodHint::ImhDigitsOnly</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="btn_cancelar_nueva_mp">
             <property name="text">
              <string>Cancelar</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_guardar_nueva_mp">
             <property name="text">
              <string>Guardar</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_entradas_mp">
      <attribute name="title">
       <string>Entradas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
           <item>
            <widget class="QLineEdit" name="le_buscar_entrada_mp">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="placeholderText">
              <string>Buscar por nombre</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableView" name="tv_entradas_list">
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
